playwright/ci-test/tests/06-resources-page.spec.ts

```diff
@@ -0,0 +1,429 @@
+import { test, expect } from '@playwright/test';
+
+let url = '/';
+
+test('resources pages', async ({ page }) => {
+  await page.goto(url);
+  await page.locator('section').filter({ hasText: 'Free and open source Spatial' }).getByRole('link').click();
+  await expect(page.locator('#sidebar').getByRole('link', { name: 'Resources', exact: true })).toBeVisible();
+  await expect(page.locator('#sidebar div').filter({ hasText: 'Resources' }).getByRole('img')).toBeVisible();
+  await page.locator('#sidebar').getByRole('link', { name: 'Resources', exact: true }).click();
+  await expect(page.getByText('Resources ¶ 🖖 Troubles with')).toBeVisible();
+  await expect(page.getByText('Troubles with installation? ¶ If you are looking for information on installing')).toBeVisible();
+  await expect(page.locator('a').filter({ hasText: /^Installation guide$/ })).toBeVisible();
+  await expect(page.getByText('QGIS has a lot of')).toBeVisible();
+  await expect(page.getByText('You will find documentation')).toBeVisible();
+  await expect(page.getByText('Get involved and help us')).toBeVisible();
+  await expect(page.locator('p').filter({ hasText: 'Get involved and help us' }).getByRole('link')).toBeVisible();
+  await expect(page.getByText('QGIS 3.28 QGIS 3.34 QGIS')).toBeVisible();
+  await expect(page.getByText('QGIS 3.28', { exact: true })).toBeVisible();
+  await expect(page.getByText('QGIS 3.34', { exact: true })).toBeVisible();
```

Comment:
3.28, 3.34 will be replaced with latest/long-term versions. automation for /resources/hub/ is not done yet, but will be soon. I suggest to remove checking the exact QGIS version for now

Reply:
Okay, I've removed all checks against versions